### ISSUE TITLE
您好，运行NER的结果出现了偏差，求助一下

### ISSUE THREAD

**Issue by RX-00-Cat** [CLUEbenchmark/CLUENER#66]
是这样的，代码跑完后，并没有出现正确的标签结果（如下） ，请问应该怎么解决，非常感谢！

D:\PythonProject\EasyBert-NER\NER\models\layers\crf.py:297: UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at  ..\aten\src\ATen\native\TensorCompare.cpp:255.)
  score = torch.where(mask[i].unsqueeze(-1), next_score, score)
[Predicting] 10/10 [==============================] 65.9ms/step{'text': '四川敦煌学”。近年来，丹棱县等地一些不知名的石窟迎来了海内外的游客，他们随身携带着胡文和的著作。', 'label': {'scene': {'县': [[13, 13]]}, 'government': {'内': [[28, 28]]}}}
{'text': '尼日利亚海军发言人当天在阿布贾向尼日利亚通讯社证实了这一消息。', 'label': {'government': {'尼': [[16, 16]]}, 'organization': {'': [[27, -1]]}}}
{'text': '销售冠军：辐射3-Bethesda', 'label': {'scene': {'射': [[6, 6]]}, 'book': {'t': [[11, 11]]}}}
{'text': '所以大多数人都是从巴厘岛南部开始环岛之旅。', 'label': {'book': {'岛': [[11, 11], [17, 17]]}}}
{'text': '备受瞩目的动作及冒险类大作《迷失》在其英文版上市之初就受到了全球玩家的大力追捧。', 'label': {'government': {'》': [[16, 16]]}, 'organization': {'': [[38, -1]]}}}
{'text': 'filippagowski：14岁时我感觉自己像梵高', 'label': {'government': {'岁': [[16, 16]]}, 'organization': {'': [[23, -1]]}}}
{'text': '央视新址文化中心外立面受损严重', 'label': {}}
{'text': '单看这张彩票，税前总奖金为5063992元。本张票面缩水后阿森纳的结果全部为0，斯图加特全部为1，', 'label': {'government': {'森': [[30, 30]], '，': [[39, 39]]}}}
{'text': '你会和星级厨师一道先从巴塞罗那市中心兰布拉大道的laboqueria市场的开始挑选食材，', 'label': {'government': {'会': [[1, 1]], '中': [[16, 16]], 'r': [[31, 31]]}}}
{'text': '波特与凤凰社》的率队下更加红火。乘着7月的上升气流，《发胶》、《辛普森一家》、《谍影憧憧ⅲ》', 'label': {'organization': {'': [[44, -1]]}}}
